wordpress-mobile/WordPress-iOS
# Today widgets: revert translations fix

Fixes #n/a
Ref: https://github.com/wordpress-mobile/WordPress-iOS/pull/15185

This reverts the fix for the Today widgets translations as it appears it increased the app bundle size by 12MB. 😞 

The changes:
- `WordPressTodayWidget` - restored localization files.
- `WordPress\Resources\Localizable.strings` - removed from 3 widget targets.

<img width="259" alt="Screen Shot 2020-11-02 at 2 25 49 PM" src="https://user-images.githubusercontent.com/1816888/97920862-560bcf80-1d17-11eb-9fa3-1c9d85d491a5.png">

---
_Tip_: sometimes the widgets get... confused... and will display an 'unable to load' message. You can fix this by either:
- Build the widget scheme first, then run the `WordPress` scheme.
- Run the widget scheme directly.

<img width="294" alt="Screen Shot 2020-10-28 at 3 21 02 PM" src="https://user-images.githubusercontent.com/1816888/97498136-43695300-1931-11eb-93ce-81dc909f8622.png">

**To test:**

- Run the app.
- Add the widgets to the Today View.
- Change the device language.
- Verify the text is not translated.

![unconfigured](https://user-images.githubusercontent.com/1816888/97920969-7d629c80-1d17-11eb-8369-c7c590425873.jpg)

- In the app, set a site to use for the Widgets. (Stats > Widgets > Use this site)

![widget_site](https://user-images.githubusercontent.com/1816888/97496924-524f0600-192f-11eb-939c-a16d7708c289.jpg)

- Return to the Today View. 
- Verify:
  - The All Time and This Week widgets are not translated.
  - The Today widget is translated.

| ![english](https://user-images.githubusercontent.com/1816888/97921256-e8ac6e80-1d17-11eb-82da-1c831e9d0952.jpg) | ![spanish](https://user-images.githubusercontent.com/1816888/97921263-eea24f80-1d17-11eb-8904-6b226040af79.jpg) |
|--------|-------|

PR submission checklist:
